utility/src/main/java/org/oppia/android/util/logging/EventBundleCreator.kt

```diff
@@ -557,6 +565,30 @@ class EventBundleCreator @Inject constructor(
         store.putNonSensitiveValue("question_name", questionName)
       }
     }
+
+    /** The [EventActivityContext] corresponding to [FeatureFlagEventContext]s. */
+    class FeatureFlagContext(
+      activityName: String,
+      value: FeatureFlagEventContext
+    ) : EventActivityContext<FeatureFlagEventContext>(activityName, value) {
+      override fun EventLog.FeatureFlagContext.storeValue(store: PropertyStore) {
+        store.putSensitiveValue("uuid", uniqueUserUuid)
+        store.putSensitiveValue("session_id", sessionId)
+        store.putNonSensitiveValue("feature_flags", featureFlagsList)
+      }
+    }
+
+    /** The [EventActivityContext] corresponding to [FeatureFlagItemEventContext]s. */
+    class FeatureFlagItemContext(
+      activityName: String,
+      value: FeatureFlagItemEventContext
+    ) : EventActivityContext<FeatureFlagItemEventContext>(activityName, value) {
+      override fun EventLog.FeatureFlagItemContext.storeValue(store: PropertyStore) {
+        store.putSensitiveValue("flag_name", flagName)
+        store.putSensitiveValue("enabled_state", flagEnabledState)
+        store.putSensitiveValue("sync_status", flagSyncStatus)
```

Comment:
These are non-sensitive values

Reply:
The method here has been changed to non-sensitive values.